_data/projects/ieee-com-soc-new-project.yml

```
@@ -0,0 +1,12 @@
+name: Annual-Yearbook
+desc: A yearbook which records the precious memories of University students to add their messages in their yearbook.
+site: https://memories.ieeecsvitc.com
+tags:
+  - typescript
+  - javascript
+  - web
+  - yearbook
+  - hacktoberfest
+upforgrabs:
+  name: Annual Yearbook
+  link: https://github.com/ComputerSocietyVITC/Annual-Yearbook
```

:art: this sections should link to the label and URL for a list of issues - should we use the `hacktoberfest` label here?

```suggestion
  name: hacktoberfest
  link: https://github.com/ComputerSocietyVITC/Annual-Yearbook/labels/hacktoberfest
```

---

I am not sure if it should or shouldn't be there . I felt it was relevant to the repository.